### PR TITLE
SAVAS-Lesson 34 Exercise was completed

### PR DESCRIPTION
**Pull request by savaspamuk**

### CODE REVIEW

**Review by annasvst — COMMENTED** [review]
Hi Savaş, I have one minor comment about the logic but otherwise looks good. It would be nice to add some styling to this application (we don't do that in lessons to have time to cover other topics) so you can add it as another project to your portfolio when it's complete. But it's up to you. Let me know if you don't want to add styles here so I'll approve after the mentioned issue is solved :)

File: JS/Lesson34/savas/my-app/src/components/NewRecipe/index.jsx [hunk 1]
```diff
@@ -13,65 +13,159 @@ export const NewRecipe = () => {
...
+    const measuresArray = measuresInput.split(",").map((item) => item.trim());
+
+    for (let i = 0; i < 20; i++) {
+      if (i < measuresArray.length) {
+        data[`strMeasure${i + 1}`] = measuresArray[i];
+      } else {
+        data[`strMeasure${i + 1}`] = "";
+      }
+    }
+    console.log(data);
```
[the remark this answers is not in the document]
Comment (savaspamuk) [reply JS/Lesson34/savas/my-app/src/components/NewRecipe/index.jsx:49]: Actually, I couldn't find the way, but searching the Internet, I found this solution :)

**Comment by savaspamuk** [comment]
Hi Anna, if we are going to continue this project, of course I would like to style it. To be honest, I haven't done much styling yet because I thought we were working on a new file every lesson :)

**Review by annasvst — APPROVED** [review]
Nice, the form works according to the requirements now. :) 
In real world, you'd need to make sure that Instructions, ingredients and measures are provided in a correct format which can be really tricky. 
It would require trimming inputs, using Regular expressions and, of course, doing a server side validation to ensure that a low quality or malicious data is stored in a database.

No need to do anything now but I want you to be aware of it  as you're going to be a developer :)

**Pull request merged**